.editorconfig

```diff
@@ -0,0 +1,6 @@
+[*]
```

Comment:
Not sure that we want the VisualStudio code editor config submitted to the codebase.  I for one am using IntelliJ so it won't make any difference to me, so if the goal is consensus on styling I think we would need to stick with stylelint rules or something similar.

I could be way off base and if I am feel free to tell me so ;P

Reply:
[Editorconfig](https://editorconfig.org/) is supported by almost all editors. It sets a baseline.